javacutil/src/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -1267,4 +1280,16 @@ public static Symbol findFunction(Tree tree, ProcessingEnvironment env) {
         Types javacTypes = Types.instance(ctx);
         return javacTypes.findDescriptorSymbol(((Type) typeOf(tree)).asElement());
     }
+
+    /**
+     * finds the corresponding functional interface method for a lambda expression
```

Comment:
What is the difference to `findFunction` above? The description and implementation look very similar.